doc/sphinx/source/conf.py

```diff
@@ -48,9 +48,9 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'chpldoc'
+project = u'Chapel Documentation'
 
-author_text = os.environ.get('CHPLDOC_AUTHOR', 'AUTHOR TEXT')
+author_text = os.environ.get('CHPLDOC_AUTHOR', 'Cray, Inc')
```

Comment:
This might work fine, but I think we still want to set CHPLDOC_AUTHOR somewhere.